src/database/interface.rs

```diff
@@ -319,14 +323,37 @@ pub fn db_unvaulted_vaults(
         ],
         |row| {
             let db_vault: DbVault = row.try_into()?;
-            let unvault_tx: Vec<u8> = row.get(13)?;
+            let unvault_tx: Vec<u8> = row.get(15)?;
             let unvault_tx = UnvaultTransaction::from_psbt_serialized(&unvault_tx)
                 .expect("We store it with as_psbt_serialized");
 
             Ok((db_vault, unvault_tx))
         },
     )
 }
+
+/// Get the txid of the Unvault transaction of the vaults whose Unvault transaction is being spent
+/// but it was not yet marked as confirmed.
+pub fn db_unconfirmed_unvault_txids(db_path: &Path) -> Result<Vec<Txid>, DatabaseError> {
```

Comment:
It could use a better name, it seems you're giving the txids of `unvaulting` vaults...

Reply:
`db_unvaulted_vaults_without_first_bh`

Too long, but you get what it does

Reply:
Went with a name similar to your suggestion:
```diff
diff --git a/src/database/actions.rs b/src/database/actions.rs
index cbdff5b..323706c 100644
--- a/src/database/actions.rs
+++ b/src/database/actions.rs
@@ -1596,7 +1596,7 @@ mod test {
             .first_stage_tx_blockheight
             .is_none());
 
-        let txids = db_unconfirmed_unvault_txids(&db_path).unwrap();
+        let txids = db_txids_unvaulted_no_bh(&db_path).unwrap();
         assert_eq!(txids.len(), 1);
         assert_eq!(txids[0], unvault_txid);
     }
diff --git a/src/database/interface.rs b/src/database/interface.rs
index bdcd009..580c9ba 100644
--- a/src/database/interface.rs
+++ b/src/database/interface.rs
@@ -334,7 +334,7 @@ pub fn db_unvaulted_vaults(
 
 /// Get the txid of the Unvault transaction of the vaults whose Unvault transaction is being spent
 /// but it was not yet marked as confirmed.
-pub fn db_unconfirmed_unvault_txids(db_path: &Path) -> Result<Vec<Txid>, DatabaseError> {
+pub fn db_txids_unvaulted_no_bh(db_path: &Path) -> Result<Vec<Txid>, DatabaseError> {
     db_query(
         db_path,
         "SELECT ptx.txid FROM vaults INNER JOIN presigned_transactions as ptx \

```